tests/mail/test_sendtestemail.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.core import mail
+from django.core.management import call_command
+from django.core.management.base import CommandError
+from django.test import SimpleTestCase
+from django.utils.six import StringIO
+
+
+class SendTestEmailManagementCommand(SimpleTestCase):
+    """
+    Test the sending of a test email using management command "sendtestemail"
+    """
+
+    def test_send_test_email(self):
+        """
+        Test that mail is sent with the correct subject and recipient.
+        """
+        new_io = StringIO()
+        recipient = "joe@somewhere.org"
+        call_command("sendtestemail", recipient, stdout=new_io)
+        self.assertEqual(len(mail.outbox), 1)
+        mail_message = mail.outbox[0]
+        self.assertEquals(mail_message.subject[0:15], 'Test email from')
```

Comment:
I'm about to commit this, but for future reference note that assertEquals is a deprecated alias and should be assertEqual